test/WebSites/RazorWebSite/Services/TestBodyTagHelperComponent.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace RazorWebSite
+{
+    public class TestBodyTagHelperComponent : TagHelperComponent
+    {
+        public override int Order => 1;
+
+        public override Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
+        {
+            if (context.TagName.Equals("body", StringComparison.Ordinal) && output.Attributes.ContainsName("inject"))
```

Comment:
Always prefer using the static `string.Equals(a, b, comparison)` method. That way you never have to worry about `null` on the LHS.